webapp/src/main/webapp/src/app/components/day-invocations-bar-chart/day-invocations-bar-chart-component.ts

```diff
@@ -14,93 +14,131 @@
  * limitations under the License.
  */
 import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
-import { Router } from "@angular/router";
+import { Router } from '@angular/router';
 import { Observable } from 'rxjs';
 
 import { DailyInvocations } from '../../models/metric.model';
 
 import * as d3 from 'd3';
-//import * as d3 from 'd3';
+// import * as d3 from 'd3';
 
 // Thanks to https://github.com/onokumus/metismenu/issues/110#issuecomment-317254128
-//import * as $ from 'jquery';
+// import * as $ from 'jquery';
 declare let $: any;
 
 @Component({
-  selector: 'day-invocations-bar-chart',
+  selector: 'app-day-invocations-bar-chart',
```

Comment:
Why this renaming again? As the scope of this component is not the global app, it doesn't make sense to me

Reply:
You can check [angular style guide ](https://angular.io/guide/styleguide#component-custom-prefix) for more details